scripts/generate-readmes.js

```diff
@@ -0,0 +1,30 @@
+const { readFileSync } = require("fs");
```

Comment:
Need to do lint fixes. You can do `yarn lint -fix`

Reply:
Done